v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/ExtractionPipeline.scala

```diff
@@ -36,12 +36,31 @@ object ExtractionPipeline {
   def main(rawArgs: Array[String]): Unit = {
     val (pipelineContext, parsedArgs) = ContextAndArgs.typed[Args](rawArgs)
 
+    val inputDir = parsedArgs.inputDir
+    val outputDir = parsedArgs.outputDir
+
     // extract and convert TSVs to Msg, transform Msg and then save Msg
     // FrequencyAnalysis
+    convertAndWrite(pipelineContext, inputDir, outputDir).close()
+    () // return Unit type
+  }
+
+  /**
+    * Convert V2F TSVs to Msg and perform necessary transformations.
+    *
+    * @param pipelineContext the ScioContext in which to run this pipeline.
+    * @param inputDir the directory from which to read data.
+    * @param outputDir the directory in which to write data.
+    */
+  def convertAndWrite(
+    pipelineContext: ScioContext,
+    inputDir: String,
+    outputDir: String
+  ): ScioContext = {
     val faExtractedAndConverted = V2FExtractionsAndTransforms.extractAndConvert(
       FrequencyAnalysis,
       pipelineContext,
-      inputDir = parsedArgs.inputDir,
+      inputDir = inputDir,
       relativeFilePath = "**.csv"
```

Comment:
Now that we have tests, can you check if everything still works if we hard-code `**.csv` as the relative file path for every case? I don't think there's a meaningful difference between the various options we're passing around.

Reply:
`**.csv` for all of them results in tests passing 👍  will change